pkg/controller/ipam/validate.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Antrea Authors
+// Copyright 2025 Antrea Authors.
```

Comment:
we don't usually update this for existing files?

Reply:
updated